packages/mjml-core/src/createComponent.js

```diff
@@ -72,6 +72,20 @@ class Component {
 
   renderMJML(mjml, options = {}) {
     if (typeof mjml === 'string') {
+      if (mjml.indexOf('<mjml>') === -1) {
```

Comment:
I'm not sure that we need that if here ? 

We can just wrap it into a "safe" tag like `<fragment>` (to keep React-ish vocabulary) and just map over children as you do l 84.

It should be way simpler and prevent later `find`

Reply:
I'll give that a try

Reply:
i'm not sure if we have any "safe" tag like this that is parseable and doesn't add anything
but you could juste wrap in `mj-body`, then you don't need the `find` because your `partialMjml` will already be the body

Reply:
I think it's just to ensure that we use a known vocabulary `fragment` is a known & used term instead of mj-body which would make you think that you're starting a new document 🤔

Reply:
It's working with `fragment`. I like that it's not an existing `mj-` component so it's clear that we're only wrapping for the children.